keras_dqn/agent.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from keras.utils.np_utils import to_categorical
+
+
+class Agent(object):
+    def __init__(self, model, target_model=None, target_swap=None):
```

Comment:
network will be just keras models... to avoid creating an extra class, let us put everything into the agent. What do you think?